src/Services/EctsScrapper/utils/scrappedEctsSubjects.ts

```diff
@@ -0,0 +1,81 @@
+import axios from 'axios';
+import cheerio from 'cheerio';
+
+export const scrappedEctsSubjects = async (
+    finalSubject: {
+        name: string;
+        url: string;
+        degree: string;
+    }[]
+) => {
+    const recruitment = finalSubject
+        .filter((el, i) => {
+            return el.name !== 'Informatyka' && el.name !== 'Matematyka';
```

Comment:
What do You think about moving these names to a constant at the top of this file?